packages/ai-chat/src/common/chat-model.ts

```diff
@@ -601,6 +601,10 @@ class ChatResponseImpl implements ChatResponse {
         return this._content;
     }
 
+    addContents(contents: ChatResponseContent[]): void {
+        contents.forEach(c => this.addContent(c));
```

Comment:
does it make sense to take this shortcut?
each add call will fire an change event which will lead to many change events which we maybe don't want?

Reply:
Good question, I was mainly trying to avoid the use of this when I had to add it a second time. The first time was already here:
https://github.com/eclipse-theia/theia/pull/14196/files/9a7013848910ae57dcb77e30cde9a3410bc95d0a#diff-b2169b1bd3b36aa51d71862eb11d8ca6ea57f214898af862f0cf6777656778bdL301

I assume, sending just one event would be fine. WDYT?